### ISSUE TITLE
Installation error on 32 bit Linux machines

### ISSUE THREAD

**Issue by BoundaryValueProblems** [jheinen/GR.jl#154]
@jheinen , it would be great if you could modify deps/build.jl so that one can install GR.jl on 32 bit linux machines without any problems. Currently, the following error occurs when it tries to build GR:
```
ERROR: LoadError: failed process: Process(`curl -g -L -f -o downloads/gr-latest-
Linux-i686.tar.gz http://gr-framework.org/downloads/gr-latest-Linux-i686.tar.gz`
, ProcessExited(22))
```
This is because there is only gr-latest-Linux-i386.tar.gz but not gr-latest-Linux-i686.tar.gz, and many 32 bit machines are identified as ```i686``` machines rather than ```i386``` machines. A potential solution would be either to put the latter file on http://gr-framework.org or to revise deps/build.jl in such a way that if the machine's ```arch``` via ```Sys.ARCH``` is ```i686``` then download ```i386``` version of the gzipped tar file.
Thanks a lot!!

**Comment by jheinen** [comment]
Thanks for the hint: I fixed this in https://github.com/jheinen/GR.jl/commit/2291c55e1de8fdb8f56fcd19d2705da7d5a19a9c

**Comment by BoundaryValueProblems** [comment]
@jheinen , I updated my copy of GR.jl to the current master branch, i.e., https://github.com/jheinen/GR.jl/commit/2291c55e1de8fdb8f56fcd19d2705da7d5a19a9c . When I ran ```pkg> build GR```, it finished building GR, but it didn't download the GR library from gr-framework.org, and didn't build the library files under ```~/.julia/packages/GR/dNvPY/deps/gr```. The file ```build.log``` is there, but it's empty. On the other hand, somehow, GR.jl is working, i.e., generating plots, passing the tests, etc. I'm very curious why.
